### PR TITLE
proposed to re45 and p2016

### PR DESCRIPTION
**Pull request by SinB-XoXo**
dmg reduction for all ranges

if approved, will update retouched.nut

### CODE REVIEW

File: keyvalues/scripts/weapons/mp_weapon_autopistol.txt [hunk 1]
```diff
@@ -0,0 +1,583 @@
...
+	{
+		"ammo_default_total"							"200"
+		"ammo_stockpile_max"							"200"
+		"ammo_no_remove_from_stockpile"					"1"
+		"ammo_min_to_fire"								"1"
+
+		"aimassist_adspull_weaponclass"					"precise"
+
+		// Damage - When Used by Players
+		"damage_near_value"   							"20"
```
Comment (fvnkhead) [inline keyvalues/scripts/weapons/mp_weapon_autopistol.txt:109]: Remember the comments: 
```suggestion
		"damage_near_value"   							"20" // CHANGE/NERF: original -> 20 (original -> 5 BTK)
```

File: keyvalues/scripts/weapons/mp_weapon_autopistol.txt [hunk 2]
```diff
@@ -0,0 +1,583 @@
...
+			"reloadempty_time"							"*0.7"
+			"reloadempty_time_late1"					"*0.7"
+		}
+		extended_ammo
+		{
+			"ammo_stockpile_max"						"375"
+			"ammo_clip_size"   							"25"
+			"ammo_default_total"						"375"
+		}
+		silencer //HACK JFS: Doesn't get applied on amped weapons. See bug 170460
```
Comment (fvnkhead) [inline keyvalues/scripts/weapons/mp_weapon_autopistol.txt:438]: This is the silencer section

Comment (SinB-XoXo) [reply keyvalues/scripts/weapons/mp_weapon_autopistol.txt:438]: i did not update the silencer section. didnt cross my mind.
but with what im reading, the values are "--5" . "--1" and "--3" based on respective ranges.
the double negative sign i would assume is a deduction on the regular damage values from mp_base (which we already nerfed based on the PR)

i assume the comments are also BTK that are based on the original values.

**Comment by SinB-XoXo** [comment]
closing PR. needing to redo

File: README.md [hunk 3]
```diff
@@ -38,6 +38,8 @@ Change summary
 | Spitfire   | 35 -> 25 (3 -> 4 BTK) | 25 -> 20 (4 -> 5 BTK)  | 20 -> 17 (5 -> 6 BTK)  |
 | EVA-8      | 200 -> 150            | 10 (_vanilla_)         | -                      |
 | Melee      | 100 -> 70             | -                      | -                      |
+| Re45       | 20 -> 15 (4 -> 7 BTK) | 13 -> 12 (7 -> 9  BTK) | 9  -> 8 (12 -> 13 BTK) |
```
Comment (fvnkhead) [inline README.md:41]: I think we should start with +2 BTK instead of +3, or..?
Also RE-45 is the correct name

**Pull request closed without merging**